### ISSUE TITLE
 I can't add the GeckoLib Plugin

### ISSUE THREAD

**Issue by CreppyJosh** [JannisX11/blockbench#771]
Hello, I liked BlockBench too much but I have a problem, when I try to add the GeckoLib plugin I get a text that says * This plugin does not work with the version of your BlockBench *
If you refer to the version of BlockBench I have the latest which is 3.7.4 and it still does not work, I downloaded it from the official BlockBench website and there is no update to this problem, Thanks for the help.
I NEED HELP!
![Yup](https://user-images.githubusercontent.com/74800373/99862238-8ac4b700-2b5e-11eb-84ed-3b9376c0bc97.png)

**Comment by JannisX11** [comment]
From the GeckoLib discord FAQ:

The old GeckoLib 2.0 plugin is not compatible with the new blockbench 3.7 update. If you wish to use GeckoLib with the new blockbench update, you must use the GeckoLib 3.0.0 beta and the new blockbench plugin that comes along with it. Read installation instructions here: https://geckolib.com/en/latest/3.0.0/. Keep in mind that GeckoLib 3.0.0 no longer supports .java files, only bedrock models (.geo.json) 
 
To summarize: 
- If you wish to keep using .java models and GeckoLib 2.0.0, don't upgrade to blockbench 3.7. Please try to update your models and switch to GeckoLib 3.0.0 soon. 
- If you wish to start using .geo.json models and participate in the GeckoLib 3.0.0 beta, simply uninstall the old plugin and install the new one using these instructions: https://geckolib.com/en/latest/3.0.0/
